api/urls_v2.py

```diff
@@ -59,6 +59,9 @@ class AplusRouter(ExtendedDefaultRouter):
     courses.register(r'resultsdata',
                      exercise.api.csv.views.CourseResultsDataViewSet,
                      basename='course-resultsdata')
+    courses.register(r'resultsdatabest',
```

Comment:
Could add a comment that this is a temporary experimental fix, and the API endpoint should not be used by external applications.